### ISSUE TITLE
Missing whenNotPaused modifier on approve

### ISSUE THREAD

**Issue by mgcolburn** [dforce-network/dToken#18]
Severity: Low

### Description
`DToken`'s `approve` function is missing a `whenNotPaused` modifier. The other entry points for users have the modifier so it appears to be an oversight to omit it from this function.

```
DToken.approve(address,uint256) (contracts/DToken.sol#866-870) should use the Pausable modifier. It changes:
        -DToken.allowance (contracts/DToken.sol#42)
```

This issue was identified by Slither.

### Recommendation
Short term, add the missing `whenNotPaused` modifier.

Long term, use [crytic.io](crytic.io)—Crytic catches this bug. This will help ensure that vulnerabilities are not introduced into the codebase.

**Comment by Donald-Nobel** [comment]
Will add it.

**Comment by bamboo-seven** [comment]
Fixed in 792bb333a1770e8da6a67c3730246f920049153e
